action.yaml

```
@@ -20,6 +20,8 @@ inputs:
     description: "Buildkit version to use with Copa"
   copa-version:
     description: "Copa version to use"
+  output-file:
```

looks like we weren't consistent before, let's match the cli output

```suggestion
  output:
```